appnav/src/main/kotlin/io/element/android/appnav/room/joined/JoinedRoomLoadedFlowNode.kt

```diff
@@ -95,6 +98,7 @@ class JoinedRoomLoadedFlowNode @AssistedInject constructor(
             },
             onDestroy = {
                 Timber.v("OnDestroy")
+                activeRoomHolder.removeRoom(inputs.room.sessionId)
```

Comment:
I still believe that this would be useful to provide the `roomId` to the `removeRoom` api, to double check that we are removing the correct room, and not the last added one.

Also one question, what will happen if there is an active call in the room, but the user closes the timeline? I guess the only issue is that there will be no cache, so not a real problem.

Reply:
> I still believe that this would be useful to provide the roomId to the removeRoom api, to double check that we are removing the correct room, and not the last added one.

Makes sense.

> Also one question, what will happen if there is an active call in the room, but the user closes the timeline? I guess the only issue is that there will be no cache, so not a real problem.

I'd say nothing really. A call needs the sync to work and an active timeline to *start* the call, but if it's already running there should be no issues, at least with the current features.